projects/scion/workbench/src/lib/common/dom.util.ts

```diff
@@ -32,6 +33,22 @@ export function setStyle(element: HTMLElement | ElementRef<HTMLElement>, styles:
   });
 }
 
+/**
+ * Sets the given attribute(s) on the given element.
+ * To remove an attribute, set its value to `null`.
+ */
+export function setAttribute(element: HTMLElement | ElementRef<HTMLElement>, attributes: {[name: string]: any | null}): void {
```

Comment:
```suggestion
export function setAttribute(element: HTMLElement | ElementRef<HTMLElement>, attributes: {[name: string]: unknown | null}): void {
```
?

Reply:
Should be a `string`. I also changed the other occurrences (`setStyle`, `setCssVariable`).